src/core/thread/topology.hpp

```diff
@@ -455,6 +455,19 @@ class Child : public Neighbor
      */
     otError RemoveIp6Address(const Ip6::Address &aAddress);
 
+    /**
+     * This method retrieves the address with specific prefix.
+     *
+     * @param[in]   aPrefix       A reference to specific prefix.
+     * @param[in]   aPrefixLen    The length of the prefix @p aPrefix.
```

Comment:
Is the `aPrefixLen` in terms of bits or bytes?  I suggest using bits to be more flexible and consistent with `kMeshLocalPrefixLength` definitions in `Ip6::Address` class.

Reply:
We can probably remove this method with #2504.